react/game/src/constants/apiNames.js

```diff
@@ -0,0 +1,2 @@
+export const BASE_URL = 'http://localhost:4000';
```

Comment:
This URL is sensible data, and maybe u need to change it without changing the code depending the environment that u would use. it must be located in `.env` file, and that file shouldn't be commited. Check this https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d